### PR TITLE
OC-12780: solve conflicts, more code change after testing

### PR DESCRIPTION
**Pull request by Taoli2018**

### CODE REVIEW

File: core/src/main/java/org/akaza/openclinica/dao/managestudy/StudyEventDAO.java [hunk 1]
```diff
@@ -1277,7 +1277,7 @@ public boolean isThisRepeatingEventScheduledMoreThanOneTime(int studyId,int sed_
 
         if (it.hasNext()) {
             Integer count = (Integer) ((HashMap) it.next()).get("count");
-            if(count.intValue() > 1) {
+            if(count.intValue() > 0) {
```
Comment (svadla-oc) [inline core/src/main/java/org/akaza/openclinica/dao/managestudy/StudyEventDAO.java:1280]: I don't think we want to change this logic. Perhaps the calling method is not handling the result of this method properly

**Review by svadla-oc — CHANGES_REQUESTED** [review]

File: core/src/main/java/org/akaza/openclinica/dao/managestudy/StudyEventDAO.java (hunk 1 above)
Comment (Taoli2018) [reply core/src/main/java/org/akaza/openclinica/dao/managestudy/StudyEventDAO.java:1280]: I double check the code, count >0 means that we found at least one subject who has been scheduled this repeating event more than once, count >1 means that we found at least two subjects who has been scheduled this repeating event more than once, so should change the logic here

Comment (svadla-oc) [reply core/src/main/java/org/akaza/openclinica/dao/managestudy/StudyEventDAO.java:1280]: Sounds good.

**Comment by svadla-oc** [comment]
👍

**Pull request merged**